webapi/webui.py

```diff
@@ -7,12 +7,16 @@
 from deluge.ui.web.json_api import export as export_api
 from deluge.plugins.pluginbase import WebPluginBase
 
+from common import get_resource
+
 LOGGER = logging.getLogger(__name__)
 
 
 
 class WebUI(WebPluginBase):
 
+    scripts = [get_resource("webapi.js")]
```

Comment:
Single quotes for string literals please.

Reply:
sorry forgot that one !